corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -50,6 +50,7 @@
 
 class Command(BaseCommand):
     help = "This command helps migrate CCHQ applications from Bootstrap 3 to Bootstrap 5."
+    skip_all = False
```

Comment:
I think I'm a little confused by your question? It's providing a default for [this line](https://github.com/dimagi/commcare-hq/pull/34473/files#diff-bb2bc01457c634b11d5f6cf9a5f5cc7686302ea13b8c1dc1fbb95f8bfd11c96aR126)
so that [this method](https://github.com/dimagi/commcare-hq/pull/34473/files#diff-bb2bc01457c634b11d5f6cf9a5f5cc7686302ea13b8c1dc1fbb95f8bfd11c96aR121) before it doesn't throw an error.

Reply:
Got it! Thanks for explaining.
Will moving this https://github.com/dimagi/commcare-hq/blob/4b98839209a4e933bebe6e6af37c6c3c93d003fc/corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py#L126 above(maybe in line 92) should have the same effect?

Reply:
no, because if `skip_all` is `True` then [this](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py#L498-L500) will automatically apply the commit and I don't want to add another `skip_all` check

Reply:
I see. Thanks for explaining. I was thinking that if `skip_all` is not passed by default it will be `False` as it the default value.
https://github.com/dimagi/commcare-hq/blob/4b98839209a4e933bebe6e6af37c6c3c93d003fc/corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py#L74

And then the management command's entry point would be the `handle` function so it would be available for the class. I might have not understood it properly and it was not a blocker just something that I wanted to understand.